// @aws-sdk/client-transcribe/dist-cjs/Transcribe.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Transcribe = void 0;
const smithy_client_1 = require("@smithy/smithy-client");
const CreateCallAnalyticsCategoryCommand_1 = require("./commands/CreateCallAnalyticsCategoryCommand");
const CreateLanguageModelCommand_1 = require("./commands/CreateLanguageModelCommand");
const CreateMedicalVocabularyCommand_1 = require("./commands/CreateMedicalVocabularyCommand");
const CreateVocabularyCommand_1 = require("./commands/CreateVocabularyCommand");
const CreateVocabularyFilterCommand_1 = require("./commands/CreateVocabularyFilterCommand");
const DeleteCallAnalyticsCategoryCommand_1 = require("./commands/DeleteCallAnalyticsCategoryCommand");
const DeleteCallAnalyticsJobCommand_1 = require("./commands/DeleteCallAnalyticsJobCommand");
const DeleteLanguageModelCommand_1 = require("./commands/DeleteLanguageModelCommand");
const DeleteMedicalTranscriptionJobCommand_1 = require("./commands/DeleteMedicalTranscriptionJobCommand");
const DeleteMedicalVocabularyCommand_1 = require("./commands/DeleteMedicalVocabularyCommand");
const DeleteTranscriptionJobCommand_1 = require("./commands/DeleteTranscriptionJobCommand");
const DeleteVocabularyCommand_1 = require("./commands/DeleteVocabularyCommand");
const DeleteVocabularyFilterCommand_1 = require("./commands/DeleteVocabularyFilterCommand");
const DescribeLanguageModelCommand_1 = require("./commands/DescribeLanguageModelCommand");
const GetCallAnalyticsCategoryCommand_1 = require("./commands/GetCallAnalyticsCategoryCommand");
const GetCallAnalyticsJobCommand_1 = require("./commands/GetCallAnalyticsJobCommand");
const GetMedicalTranscriptionJobCommand_1 = require("./commands/GetMedicalTranscriptionJobCommand");
const GetMedicalVocabularyCommand_1 = require("./commands/GetMedicalVocabularyCommand");
const GetTranscriptionJobCommand_1 = require("./commands/GetTranscriptionJobCommand");
const GetVocabularyCommand_1 = require("./commands/GetVocabularyCommand");
const GetVocabularyFilterCommand_1 = require("./commands/GetVocabularyFilterCommand");
const ListCallAnalyticsCategoriesCommand_1 = require("./commands/ListCallAnalyticsCategoriesCommand");
const ListCallAnalyticsJobsCommand_1 = require("./commands/ListCallAnalyticsJobsCommand");
const ListLanguageModelsCommand_1 = require("./commands/ListLanguageModelsCommand");
const ListMedicalTranscriptionJobsCommand_1 = require("./commands/ListMedicalTranscriptionJobsCommand");
const ListMedicalVocabulariesCommand_1 = require("./commands/ListMedicalVocabulariesCommand");
const ListTagsForResourceCommand_1 = require("./commands/ListTagsForResourceCommand");
const ListTranscriptionJobsCommand_1 = require("./commands/ListTranscriptionJobsCommand");
const ListVocabulariesCommand_1 = require("./commands/ListVocabulariesCommand");
const ListVocabularyFiltersCommand_1 = require("./commands/ListVocabularyFiltersCommand");
const StartCallAnalyticsJobCommand_1 = require("./commands/StartCallAnalyticsJobCommand");
const StartMedicalTranscriptionJobCommand_1 = require("./commands/StartMedicalTranscriptionJobCommand");
const StartTranscriptionJobCommand_1 = require("./commands/StartTranscriptionJobCommand");
const TagResourceCommand_1 = require("./commands/TagResourceCommand");
const UntagResourceCommand_1 = require("./commands/UntagResourceCommand");
const UpdateCallAnalyticsCategoryCommand_1 = require("./commands/UpdateCallAnalyticsCategoryCommand");
const UpdateMedicalVocabularyCommand_1 = require("./commands/UpdateMedicalVocabularyCommand");
const UpdateVocabularyCommand_1 = require("./commands/UpdateVocabularyCommand");
const UpdateVocabularyFilterCommand_1 = require("./commands/UpdateVocabularyFilterCommand");
const TranscribeClient_1 = require("./TranscribeClient");
const commands = {
    CreateCallAnalyticsCategoryCommand: CreateCallAnalyticsCategoryCommand_1.CreateCallAnalyticsCategoryCommand,
    CreateLanguageModelCommand: CreateLanguageModelCommand_1.CreateLanguageModelCommand,
    CreateMedicalVocabularyCommand: CreateMedicalVocabularyCommand_1.CreateMedicalVocabularyCommand,
    CreateVocabularyCommand: CreateVocabularyCommand_1.CreateVocabularyCommand,
    CreateVocabularyFilterCommand: CreateVocabularyFilterCommand_1.CreateVocabularyFilterCommand,
    DeleteCallAnalyticsCategoryCommand: DeleteCallAnalyticsCategoryCommand_1.DeleteCallAnalyticsCategoryCommand,
    DeleteCallAnalyticsJobCommand: DeleteCallAnalyticsJobCommand_1.DeleteCallAnalyticsJobCommand,
    DeleteLanguageModelCommand: DeleteLanguageModelCommand_1.DeleteLanguageModelCommand,
    DeleteMedicalTranscriptionJobCommand: DeleteMedicalTranscriptionJobCommand_1.DeleteMedicalTranscriptionJobCommand,
    DeleteMedicalVocabularyCommand: DeleteMedicalVocabularyCommand_1.DeleteMedicalVocabularyCommand,
    DeleteTranscriptionJobCommand: DeleteTranscriptionJobCommand_1.DeleteTranscriptionJobCommand,
    DeleteVocabularyCommand: DeleteVocabularyCommand_1.DeleteVocabularyCommand,
    DeleteVocabularyFilterCommand: DeleteVocabularyFilterCommand_1.DeleteVocabularyFilterCommand,
    DescribeLanguageModelCommand: DescribeLanguageModelCommand_1.DescribeLanguageModelCommand,
    GetCallAnalyticsCategoryCommand: GetCallAnalyticsCategoryCommand_1.GetCallAnalyticsCategoryCommand,
    GetCallAnalyticsJobCommand: GetCallAnalyticsJobCommand_1.GetCallAnalyticsJobCommand,
    GetMedicalTranscriptionJobCommand: GetMedicalTranscriptionJobCommand_1.GetMedicalTranscriptionJobCommand,
    GetMedicalVocabularyCommand: GetMedicalVocabularyCommand_1.GetMedicalVocabularyCommand,
    GetTranscriptionJobCommand: GetTranscriptionJobCommand_1.GetTranscriptionJobCommand,
    GetVocabularyCommand: GetVocabularyCommand_1.GetVocabularyCommand,
    GetVocabularyFilterCommand: GetVocabularyFilterCommand_1.GetVocabularyFilterCommand,
    ListCallAnalyticsCategoriesCommand: ListCallAnalyticsCategoriesCommand_1.ListCallAnalyticsCategoriesCommand,
    ListCallAnalyticsJobsCommand: ListCallAnalyticsJobsCommand_1.ListCallAnalyticsJobsCommand,
    ListLanguageModelsCommand: ListLanguageModelsCommand_1.ListLanguageModelsCommand,
    ListMedicalTranscriptionJobsCommand: ListMedicalTranscriptionJobsCommand_1.ListMedicalTranscriptionJobsCommand,
    ListMedicalVocabulariesCommand: ListMedicalVocabulariesCommand_1.ListMedicalVocabulariesCommand,
    ListTagsForResourceCommand: ListTagsForResourceCommand_1.ListTagsForResourceCommand,
    ListTranscriptionJobsCommand: ListTranscriptionJobsCommand_1.ListTranscriptionJobsCommand,
    ListVocabulariesCommand: ListVocabulariesCommand_1.ListVocabulariesCommand,
    ListVocabularyFiltersCommand: ListVocabularyFiltersCommand_1.ListVocabularyFiltersCommand,
    StartCallAnalyticsJobCommand: StartCallAnalyticsJobCommand_1.StartCallAnalyticsJobCommand,
    StartMedicalTranscriptionJobCommand: StartMedicalTranscriptionJobCommand_1.StartMedicalTranscriptionJobCommand,
    StartTranscriptionJobCommand: StartTranscriptionJobCommand_1.StartTranscriptionJobCommand,
    TagResourceCommand: TagResourceCommand_1.TagResourceCommand,
    UntagResourceCommand: UntagResourceCommand_1.UntagResourceCommand,
    UpdateCallAnalyticsCategoryCommand: UpdateCallAnalyticsCategoryCommand_1.UpdateCallAnalyticsCategoryCommand,
    UpdateMedicalVocabularyCommand: UpdateMedicalVocabularyCommand_1.UpdateMedicalVocabularyCommand,
    UpdateVocabularyCommand: UpdateVocabularyCommand_1.UpdateVocabularyCommand,
    UpdateVocabularyFilterCommand: UpdateVocabularyFilterCommand_1.UpdateVocabularyFilterCommand,
};
class Transcribe extends TranscribeClient_1.TranscribeClient {
}
exports.Transcribe = Transcribe;
(0, smithy_client_1.createAggregatedClient)(commands, Transcribe);
